src/set.js

```diff
@@ -0,0 +1,99 @@
+export default class Set {
+  constructor() {
+    this.length = 0
+    this.elements = []
+  }
+
+  add = (value) => {
+    if(!this.contains(value)){
+      this.elements.push(value)
+      this.length += 1
+    }
+  }
+
+  isEmpty = () => this.length === 0
+
+  contains = (value) => {
+    for (let i = 0; i < this.length; i++) {
+      if (this.elements[i] === value) {
+        return true
+      }
+    }
+    return false
+  }
+
+  remove = (value) => {
+    if(this.contains(value)){
+      let removeIndex = 0
+      for(let i = 0; i < this.length; i++){
+        if(this.elements[i] === value){
+          removeIndex = i
+        }
+      }
+      this.elements.splice(removeIndex, 1)
+      this.length -= 1
+    }
+  }
+
+  forEach = passedInFunction => {
+    const setSize = new Number(this.length)
+    for (let i = 0; i < setSize; i++) {
+      passedInFunction(this.elements[i])
+    }
+  }
+
+  size = () => this.length
+
+  union = (otherSet) => {
+    let newSet = new Set()
+    newSet = this
```

Comment:
This line could just be:
```js
let newSet = this
```
Having said that, you want to return a new `Set` object, so you would not want to do that - you should be creating a new `Set`, as you do on line 48, and adding the elements to it (which appears to be what you're doing).  With the current code, it seems as if you are mutating the current `Set`, which is not the intent of the set operations functions.